clients/src/main/java/org/apache/kafka/clients/consumer/KafkaConsumer.java

```diff
@@ -1053,7 +1053,7 @@ public void assign(Collection<TopicPartition> partitions) {
      *             function is called
      * @throws org.apache.kafka.common.errors.InterruptException if the calling thread is interrupted before or while
      *             this function is called
-     * @throws org.apache.kafka.common.errors.AuthenticationFailedException if caller's authentication fails
+     * @throws org.apache.kafka.common.errors.AuthenticationException if caller's authentication fails
```

Comment:
I noticed that there is no Javadoc for AuthenticationException and subclasses. It would be good to add some and to explain what is the difference between them. It could be done in a separate PR. cc @rajinisivaram

Reply:
Same for AuthorizationException.

Reply:
@ijuma Thank you for the review. @vahidhashemian Can you add `@see AuthenticationException/AuthorizationException` to all the javadoc entries and I will submit another PR with more docs for the exceptions. Thanks.